src/eda_server/config/enums.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright 2022 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""EDA Server enum definitions."""
+
+from enum import Enum
+
+
+class DeploymentTypeEnum(Enum):
+    LOCAL = "local"
+    DOCKER = "docker"
+    PODMAN = "podman"
+    K8S = "k8s"
+    KUBERNETES = "kubernetes"
```

Comment:
What is the difference between `k8s` and `kubernetes` ?

Reply:
JIC someone wants to spell it out. K8S was the original, IIRC.